com.github.maoschanz.DynamicWallpaperEditor.json

```
@@ -0,0 +1,39 @@
+{
+    "app-id" : "com.github.maoschanz.DynamicWallpaperEditor",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.28",
```

3.30 is out now, please test against it.

```suggestion
    "runtime-version" : "3.30",
```